presqt/api_v1/tests/views/test_target.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIRequestFactory
+
+from presqt.api_v1.views.target import TargetsList, TargetDetails
+
+
+class TestTargetsList(TestCase):
+    """
+    Test the `api_v1/targets/` endpoint's GET method.
+    """
+    def test_get_success(self):
+        """
+        Return a 200 if the GET method is successful
+        """
+        self.factory = APIRequestFactory()
+        view = TargetsList.as_view()
+        request = self.factory.get('api_v1/targets/')
+        response = view(request)
+
+        # Verify the Status Code
+        self.assertEqual(response.status_code, 200)
+        # Verify that the first dictionary in the payload's array has the correct keys
+        self.assertListEqual(list(response.data[0].keys()), ['name', 'read', 'write', 'detail'])
```

Comment:
I suppose it would be *more* explicit to use a `for` loop to verify each dictionary in the payload is correctly shaped.